src/pages/css/css3-borders-rounded-corners/index.md

```diff
@@ -3,13 +3,24 @@ title: CSS3 Borders Rounded Corners
 ---
 ## CSS3 Borders Rounded Corners
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/css/css3-borders-rounded-corners/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+With CSS3, you can give any element "rounded corners", by using the border-radius property. The value can be in any valid CSS unit.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+```css
+    .rounded-corners {
+        border-radius: 20px;
+    }
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+    .circle {
+        border-radius: 50%;
+    }
+```
+
+**Note:** The border-radius property is actually a shorthand property for the border-top-left-radius, border-top-right-radius, border-bottom-right-radius and border-bottom-left-radius properties. 
```

Comment:
Add syntax highlighting around the property names, so it's "... `border-top-left-radius`, `border-top-right-radius`, `border-bottom-right-radius`, and `border-bottom-left-radius` properties."